app/views/tasklists/index.html.haml

```diff
@@ -1,14 +1,32 @@
-%h2 Hello #{current_user.email}
+%nav.navbar.navbar-light.bg-warning
+  .container-fluid
+    .navbar-header
+      %a.navbar-brand{href:"#"} Todo List
+    %ul.nav.nav-pills
+      %li.nav-item
+        -if user_signed_in?
+          =link_to 'Sign out', destroy_user_session_path, method: 'delete', class:"nav-link"
 %br
-- if @tasklist.length
-  %h3 You have #{@tasklist.length} tasklist
-  -@tasklist.each do |tasklist|
-    =link_to tasklist.name, tasklist_tasks_path(tasklist.id)
-    %br
-
-  %br
-
-=link_to 'Create new tasklist', new_tasklist_path
-
-%br
-=link_to 'Sign Out',destroy_user_session_path,method: :delete if user_signed_in?
\ No newline at end of file
+.container
+  .row
+    .col-sm-2
+      Task lists
+      %a.float-right.fa.fa-plus{onClick: 'addTasklist()'}
+      %hr
+      #newtasklist
+      #tasklists
+        - tasklistCount = 0
+        -@tasklists.each do |tasklist|
```

Comment:
use each_with_index if you need access to index

Reply:
done